src/gameplay.js

```diff
@@ -36,31 +39,31 @@ const Gameplay = () => {
     }
 
 
-    const writeLeaderboard = () =>
-    {
+    const writeLeaderboard = () => {
```

Comment:
Thank you for adding the score posting functionality.

Reply:
No problem!